spec/openapi.yaml

```diff
@@ -53,182 +248,743 @@ paths:
+          description: A list of resources
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ApiKeyResponse'
-        401:
-          description: Unauthorized
+                $ref: '#/components/schemas/FilteredResourcesResponse'
+              example: |
+                {
+                  'apiVersion': '1.0',
+                  'data': 
+                  [
+                      {
+                        'category': 'Tutorial',
+                        'created_at': '2019-07-21 15:41:19',
+                        'downvotes': 0,
+                        'id': 2137,
+                        'languages': [],
+                        'last_updated': '2019-07-22 17:39:50',
+                        'name': 'Some, but not ALL of the knowledge',
+                        'notes': null,
+                        'paid': false,
+                        'times_clicked': 3,
+                        'upvotes': 1,
+                        'url': 'http://www.test.com'
+                      }
+                  ],
+                    'has_next': false,
+                    'has_prev': false,
+                    'number_of_pages': 1,
+                    'page': 1,
+                    'records_per_page': 20,
+                    'status': 'ok',
+                    'status_code': 200,
+                    'total_count': 1
+                }
+        404:
+          $ref: '#/components/responses/NotFound'
+    post:
+      summary: Create Resource
+      description: Creates a new learning resource entry. The required properties for a new entry are `category`, `name`, `paid`, and `url` and must be included in the request body.
+      tags:
+        - Create Resource
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateResourceRequest'
+            example: |
+              {
+                    'category': 'Tutorial',
+                    'name': 'CSS Tutorial',
+                    'paid': false,
+                    'url': 'https://www.w3schools.com/css/'
+                }
+        required: true
+      responses:
+        200:
+          description: Created resource
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Error'
+                $ref: '#/components/schemas/Resource'
+              example: |
+                    {
+                        'apiVersion': '1.0',
+                        'data': {
+                            'category': 'Tutorial',
+                            'created_at': '2019-07-21 15:41:19',
+                            'downvotes': 0,
+                            'id': 2137,
+                            'languages': [],
+                            'last_updated': '2019-07-22 16:05:53',
+                            'name': 'Some, but not ALL of the knowledge',
```

Comment:
For `Create` and `Update` routes, the response should match the response you'd expect to receive from the example request. In this case, the example request was `CSS Tutorial`, so the example response should contain that in the `name` field.